core/src/main/java/io/questdb/cairo/sql/SqlExecutionCircuitBreaker.java

```diff
@@ -124,6 +130,9 @@ public void unsetTimer() {
 
     boolean checkIfTripped(long millis, long fd);
 
+    // returns remaining processes to await
+    int finish();
```

Comment:
what is the purpose of this method? Return value is ignored, and it is doing the same job as `setCancelledFlag`